.github/workflows/pull-request.yml

```diff
@@ -33,30 +41,34 @@ jobs:
           sudo su -c "while ! test -S /run/buildkit/buildkitd.sock; do sleep 0.1; done"
           sudo chmod +777 /run/buildkit/buildkitd.sock
 
-      # A hack as GH action does not allow you to force override cache storing if there was a cache hit
-      # https://github.com/actions/cache/issues/628#issuecomment-986118455
-      - name: 🗄️ Force Save Registry Cache Per Sha
-        uses: actions/cache@v2
+      - name: 🤓 Update dazzle.yaml
+        run: |
+          pip3 install ruamel.yaml
+          python3 .github/update-dazzle-for-pr.py ${{ github.event.pull_request.number }}
```

Comment:
We often use `yq` to make changes to YAML, out of curiosity,m why not use it, instead of Python, to add the suffix?

Full disclosure, I like Python, I'm just curious.

Reply:
[Example using jq and yq](https://github.com/gitpod-io/gitpod/blob/main/.werft/jobs/build/deploy-to-preview-environment.ts#L351-L355) (granted, this yq version is older)

Reply:
Hey @kylos101 , I did explore the option of using `yq` but the query was unnecessarily complex to understand and write. I figured a python script was much simpler to write and understand.